docs/documentation/Tools/tools.md

```diff
@@ -1,46 +1,69 @@
-This page contains a listing of third-party codes that are known to interoperate with CASTEP. Some of these programs require additional interface codes/scripts to take CASTEP output as their input, etc.
+This page lists various analysis codes (supplied with the CASTEP distribution as well as some third-party codes) which interoperate with CASTEP. Some of these programs require additional interface codes/scripts to take CASTEP output as their input, etc.
 
 
 ## Pre-processing Tools
 
-* [c2x](http://www.c2x.org.uk/) reads and writes a selection of file formats which relate to DFT electronic structure codes to perform manipulations and analysis. It can interpolate densities via a combination of trilinear and Fourier interpolation, integrate densities and calculate their dipole moments, and analyse bands for inversion symmetry. It can be used to produce output formats compatible with XCrysDen, VESTA, VMD and Jmol amongst others, and can also produce 1D data in a format suitable for gnuplot.
+* [c2x](http://www.c2x.org.uk/) (also bundled with the CASTEP distribution) reads and writes a selection of file formats which relate to DFT electronic structure codes to perform manipulations and analysis. It can interpolate densities via a combination of trilinear and Fourier interpolation, integrate densities and calculate their dipole moments, and analyse bands for inversion symmetry. It can be used to produce output formats compatible with XCrysDen, VESTA, VMD and Jmol amongst others, and can also produce 1D data in a format suitable for gnuplot.
 * [Cif2Cell](https://sourceforge.net/projects/cif2cell/) is a tool to generate the geometrical setup for various electronic structure codes from a CIF (Crystallographic Information Framework) file. Also supports supercell generation and formats for visualisation tools.
 * [SeeK-path](https://www.materialscloud.org/work/tools/seekpath) is a k-path finder that provides band paths compatible with space group symmetry, and an interactive 3D visualiser.
 
-## Post-processing Tools
+## Convergence Testing
 
+* [castepconv](https://github.com/CCP-NC/castepconv) automates convergence testing of plane-wave cutoff, k-point density and fine grid scale. Based on a proforma input it prepares the files, analyses the output and generates plots.  An earlier release was shipped with CASTEP source, but this is superceded by the version on github.
```

Comment:
I suggest: 
CASTEPconv (also bundled with the CASTEP distribution) automates convergence testing of plane-wave cutoff, k-point density and fine grid scale. Based on a proforma input, it prepares the files, analyses the output and generates summary data files. A standalone version is shipped with CASTEP source. Later versions depend upon ASE and can be downloaded from github (https://github.com/CCP-NC/castepconv) and can also generate plots.